test/unit/annotation_spec.js

```diff
@@ -506,17 +512,17 @@ describe('annotation', function() {
         { ref: annotationRef, data: annotationDict, }
       ]);
 
-      var annotation = AnnotationFactory.create(xref, annotationRef,
-                                                pdfManagerMock, idFactoryMock);
-      var data = annotation.data;
-      expect(data.annotationType).toEqual(AnnotationType.LINK);
-
-      expect(data.url).toEqual(new URL('http://www.example.com/test.pdf#' +
+      AnnotationFactory.create(xref, annotationRef,
+          pdfManagerMock, idFactoryMock).then((annotation) => {
+        var data = annotation.data;
+        expect(data.annotationType).toEqual(AnnotationType.LINK);
+        expect(data.url).toEqual(new URL('http://www.example.com/test.pdf#' +
                                  '[14,{"name":"XYZ"},null,298.043,null]').href);
-      expect(data.unsafeUrl).toEqual('http://www.example.com/test.pdf#' +
+        expect(data.unsafeUrl).toEqual('http://www.example.com/test.pdf#' +
```

Comment:
Add two extra spaces of indenting to the line below to align it nicely with this line.